packages/core/src/server/generator/schema.ts

```diff
@@ -68,7 +68,7 @@ export function writeGraphqlSchemaFile(apiSchema: GraphQLSchema) {
     // getting the schema before write because somehow this fixes the validation step of codegen from codesandbox
     const schema = printSchemaWithDirectives(apiSchema)
     writeFileSync(
-      path.join(process.cwd(), '@generated', 'schema.graphql'),
+      path.join(__dirname, '..', '..', '..', '@generated', 'schema.graphql'),
```

Comment:
question: Is there a way to use that basePath here? I'm wondering if we could do something to make sure it would still work if something changed on the folders structure.

Reply:
You're correct about not depending on folder structure. I'm unsure how to put the `basePath` (or a derived path) here since `core` is outside the scope of the CLI and has no access to `withBasePath`

Reply:
You're right... The only idea I came up now was to add a comment there with this warning, to make it more explicit that if this file change paths it will break. What do you think?